src/resspect/database.py

```diff
@@ -234,6 +239,10 @@ def load_features_from_file(self, path_to_features_file: str, screen=False,
             "Bump", or "Malanchev". Default is "Bazin".
         """
 
+        if survey not in ['DES', 'LSST']:
```

Comment:
@AmandaWasserman Is this logic correct, or should we support any `survey` that we find in the `FILTER_SETS` dictionary?  i.e. `DES`, `LSST`, and `SNPCC`